src/silx/math/test/histo_benchmarks.py

```diff
@@ -109,7 +109,9 @@ def benchmark(
 
     if do_weights:
         weights = np.random.randint(
-            int_min, high=int_max, size=(ssetup.pyample_shape[0],)
+            int_min,
+            high=int_max,
+            size=(ssetup.pyample_shape[0],),  # flake8 F821 undefined name 'ssetup'
```

Comment:
That's a typo:
```suggestion
            int_min, high=int_max, size=(sample_shape[0],)
```

We can consider removing this file (for another PR), this is not a test

Reply:
fixed the typo and moved args / kwargs into a single line